getting-started/CONCEPTS/MODERATOR.md

```diff
@@ -0,0 +1,43 @@
+# Moderator
+
+The `Moderator` is responsible for reviewing the input prompt and approving the AI generated plans. It is configured when orchestrating the `Application` class. Here's an example of configuring the `OpenAIModerator`.
+
+**JS**
```

Comment:
May be nice to add in an example configuration of AzureContentSafetyModerator as well (either for one language or both)